### PR TITLE
installer for hazardItems

### PR DESCRIPTION
**Pull request by mhines-usgs**
- one for dev, which uses the latest and greatest versions of packages
- one for QA & prod, which will be locked down to today's date for now

### CODE REVIEW

File: inst/extdata/installPackagesDev.sh [hunk 1]
```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
```
Comment (thongsav-usgs) [inline inst/extdata/installPackagesDev.sh]: This SH script isn't actually needed, using the above R script would accomplish the same thing if you replaced the mran URL with the cran url. We may want to go ahead and parameterize line 95 in installPackages.R.

File: inst/extdata/installPackages.R [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+}
+
+pkgs <-
+  grep(
+    "gsplot", hazardItemsImports(lib.loc = lib),
+    value = TRUE, fixed = TRUE, invert = TRUE
+  )
+
+# all packages are held back to older, MRAN versions
+installPackages(c(pkgs, "devtools"), lib, "https://mran.microsoft.com/snapshot/2017-04-06")
```
Comment (thongsav-usgs) [inline inst/extdata/installPackages.R]: I'd add an if statement here to switch between mran and cran, something like "useLatestFromCran". Then on DEV have it call the R script after setting the useLatestFromCran variable to true to enable CRAN.

File: inst/extdata/installPackagesDev.sh (hunk 1 above)
Comment (mhines-usgs) [inline inst/extdata/installPackagesDev.sh]: OK

File: inst/extdata/installPackages.R [hunk 3]
```diff
@@ -1,6 +1,10 @@
 # Install hazardItems, production instance, prerequisite packages. Should be safe to
 # run repeatedly without additional intervention.
 
+# for dev to use CRAN instead of MRAN
+args = commandArgs(trailingOnly=TRUE)
```
Comment (mhines-usgs) [inline inst/extdata/installPackages.R]: catches the args passed in

File: inst/extdata/installPackages.R [hunk 4]
```diff
@@ -91,9 +95,9 @@ pkgs <-
     value = TRUE, fixed = TRUE, invert = TRUE
   )
 
-# all packages are held back to older, MRAN versions, unless it's dev, then the Dev Jenkins job should pass in -useCran TRUE
-if (useCran) {
-  installPackages(c(pkgs, "devtools"), lib, "https://cran.rstudio.com/")
-} else {
+# all packages are held back to older, MRAN versions, unless it's dev, then the Dev Jenkins job should use latest and greatest
+if (length(args)==0) {
```
Comment (mhines-usgs) [inline inst/extdata/installPackages.R]: if there aren't any args, then use mran

File: inst/extdata/installPackages.R [hunk 5]
```diff
@@ -91,9 +95,9 @@ pkgs <-
...
   installPackages(c(pkgs, "devtools"), lib, "https://mran.microsoft.com/snapshot/2017-04-06")  
-}
+  } else if (length(args)==1) {
```
Comment (mhines-usgs) [inline inst/extdata/installPackages.R]: if the useCran arg is there (dev) use cran = newest versions of packages

**Pull request merged**